Components/badge.css

```diff
@@ -0,0 +1,63 @@
+.badge-text-normal{
+    margin-bottom: 2rem;
+    font-size: 1.2rem;
+}
+.badge-number{
+    border-radius: 50%;
+    background-color: var(--blue-solid);
+    color: var(--white);
+}
+.badge-text-large{
+    margin-bottom: 2rem;
+    font-size: 2rem;
+}
+
+.badge-text-small{
+    margin-bottom: 2rem;
+    font-size: 0.7rem;
+}
+.normal{
+    width: 2rem;
+    height: 2rem;
+    top: -1rem;
+    left: 11rem;   
+}
+.large{
+    width: 2.5rem;
+    height: 2.5rem;
+    top: -1rem;
+    left: 17rem;
+}
+.small{
+    width: 1.2rem;
+    height: 1.2rem;
+    top: -1rem;
+    left: 6rem;
+}
+.icon-badge{
+    font-size: 2rem;  
+}
+.badge-on-icon{
+    width: 1.5rem;
+    height: 1.5rem;
+    font-size: 1rem;
+    top: -0.5rem;
+    left:1.8rem;
```

Comment:
For values less than 10px, rem is not preferred.

Reply:
done, -0.5rem changed to -8px ,thanks